lib/plugins/aws/package/compile/events/httpApi.js

```diff
@@ -94,6 +94,10 @@ class HttpApiEvents {
       Name: this.provider.naming.getHttpApiName(),
       ProtocolType: 'HTTP',
     };
+    if (this.serverless.service.provider.tags) {
+      const tags = Object.assign({}, this.serverless.service.provider.tags);
+      properties.Tags = tags;
```

Comment:
Were you able to successfully verify that it works as intended and sets proper tags for API Gateway? I'm asking because it seems like it uses a different notation than `Tags` usually do in CloudFormation

Reply:
I have tried this [commit](https://github.com/serverless/serverless/pull/8938/commits/9603e857eb1443731720d4a4faa35a153e190264) but cloudformation throwed an error saying Tags requires an object. After correcting with the latest commit was able to deploy and confirm that global tags are applied to httpApi. Please refer below links from AWS
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-api.html#cfn-apigatewayv2-api-tags
https://docs.aws.amazon.com/cli/latest/reference/apigatewayv2/create-api.html